homeassistant/components/idasen_desk/config_flow.py

```diff
@@ -64,6 +64,9 @@ async def async_step_user(
             desk = Desk(None)
             try:
                 await desk.connect(discovery_info.device, monitor_height=False)
+            except AuthFailedError as err:
+                _LOGGER.exception("AuthFailedError", exc_info=err)
```

Comment:
Using both the `exception` logging method and `exc_info` isn't needed.

We normally don't log the stack trace if the exception is known.